scripts/install-cni.sh

```diff
@@ -56,23 +56,32 @@ fi
 
 if [ -f "/host/home/kubernetes/bin/gke" ]
 then
-	cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniType#gke#g")
+  cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniType#gke#g")
 else
-	cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniType#ptp#g")
+  cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniType#ptp#g")
 fi
 
 if [ "${ENABLE_BANDWIDTH_PLUGIN}" == "true" ] && [ -f "/host/home/kubernetes/bin/bandwidth" ]
 then
-	cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniBandwidthPlugin#,{\"type\": \"bandwidth\",\"capabilities\": {\"bandwidth\": true}}#g")
+  cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniBandwidthPlugin#,{\"type\": \"bandwidth\",\"capabilities\": {\"bandwidth\": true}}#g")
 else
-	cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniBandwidthPlugin##g")
+  cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniBandwidthPlugin##g")
 fi
 
 if [ "${ENABLE_CILIUM_PLUGIN}" == "true" ]
 then
-        cni_spec=$(echo ${cni_spec:-} | sed -e "s#@cniCiliumPlugin#,{\"type\": \"cilium-cni\"}#g")
+  echo "Adding Cilium plug-in:"
```

Comment:
nit: "Adding Cilium plug-in to the CNI config." to be clear?

Reply:
Done